libs/ui/src/app_base.tsx

```diff
@@ -49,12 +52,29 @@ export function AppBase(props: AppBaseProps): JSX.Element {
 
   const [colorMode, setColorMode] = React.useState<ColorMode>(defaultColorMode);
   const [sizeMode, setSizeMode] = React.useState<SizeMode>(defaultSizeMode);
+  const [fontsLoaded, setFontsLoaded] = React.useState(false);
+
+  React.useEffect(() => {
+    /* istanbul ignore next - tested via integration tests. */
+    if (!disableFontsForTests) {
+      loadFonts();
```

Comment:
React runs `useEffect` multiple times in development, so we end up with multiple `<style>` elements being added. While this may not cause any visual issues, it's maybe not great for resource usage and keeping things close to production if we can.

<img width="914" alt="image" src="https://github.com/votingworks/vxsuite/assets/1938/0da42438-735c-4702-8e11-7b41286a5fb8">

Reply:
Oh, just seeing this! Thanks for the callout - didn't know about the multiple `useEffect` invocations in dev. Will add a cleanup then, to address the dev weirdness